devenv/local/docker-compose/Dockerfile.btc

```diff
@@ -0,0 +1,7 @@
+FROM debian:bookworm
+
+COPY --from=dobtc/bitcoin:25.1 /opt/bitcoin-*/bin /usr/local/bin
```

Comment:
Now the miner does use `bitcoin-cli` to interact with the node, which I agree is much better! And since these two services (miner and bitcoin node) use the same image, I think the image building cost should be minimal, right?

Reply:
Oh yeah that's right. And yeah it's pretty minimal. And more importantly, it seems annoying to change. I wouldn't worry about it.